benches/run-benchmark.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Runs a benchmark using drun. The script assumes drun is available on the caller's path.
+set -euo pipefail
+
+BENCH_NAME=$1
+FILE=mktemp
+
+if ! type "drun" > /dev/null; then
+  echo "drun is not installed. Please add drun to your path from commit d35535c96184be039aaa31f68b48bbe45909494e."
```

Comment:
It would be great if we could run these benchmarks on each push request and push the outcome as a comment.
Or at least have a CI action that runs the benchmarks.

Reply:
Absolutely. We have a [separate ticket for that](https://dfinity.atlassian.net/browse/EXC-1406), which I'll take care as part of the on-going stable structures feature.